python_apps/libretime_watch/libretime_watch/start_watching.py

```diff
@@ -59,35 +59,39 @@ def connect_database():
 conn=connect_database()
 cur = conn.cursor()
 try:
-   cur.execute ("SELECT id,directory from cc_music_dirs where type = 'watched'")
-   row = cur.fetchone()
-   id = row[0]
-   watch_dir = row[1]
-   cur.close()
+  cur.execute ("SELECT id,directory from cc_music_dirs where type = 'watched'")
+  rows = cur.fetchall()
+  cur.close()
 except:
-   logging.critical("Can't get directory for watching")
-   exit()
-
-#message = { 'cmd' : 'rescan_watch', 'id' : '34', 'directory' : '/srv/airtime/watch/'}
-message = { 'cmd' : 'rescan_watch', 'api_key' : str(config['api_key']), 'id' : str(id), 'directory' : str(watch_dir)}
-
-json_encoded = json.dumps(message)
-
-# user/password rabbitmq
-credentials=pika.credentials.PlainCredentials(config["rm_user"],config["rm_pass"])
-# connect to rabbitmq
-connection = pika.BlockingConnection(pika.ConnectionParameters(host=config["rm_host"],
-            virtual_host=config["rm_vhost"],credentials=credentials))
-channel = connection.channel()
-
-# declare exchange
-channel.exchange_declare(exchange=EXCHANGE,type=EXCHANGE_TYPE,durable=True, auto_delete=True )
-
-# .. and send message
-channel.basic_publish(exchange=EXCHANGE,
-                      routing_key=ROUTING_KEY,
-                      body=json_encoded)
-# close rabbitmq
-connection.close()
-logging.info("Triggered Watching Folders Scan")
-# TODO multiple watching directories
+  cur.close()
+  logging.critical("Can't get directory for watching")
+  exit()
```

Comment:
This should exit with a non-zero code to indicate failure

Reply:
Fixed in ab13736